src/main/resources/db/migration/V011__Schema.sql

```diff
@@ -0,0 +1,17 @@
+CREATE TABLE user_deletion_audit_old (
```

Comment:
I was thinking of something more straightforward:

1. 1.ALTER TABLE user_deletion_audit RENAME TO user_deletion_audit_old;
2. Create the new table

It's also a good practice to wrap the operation in a transaction BEGIN; COMMIT;

Reply:
Addressed. As for the transaction, flyway does its own transaction management and is not advisable to use explicit transactions in migration script.